version/version.go

```diff
@@ -125,7 +125,9 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, fmt.Sprintf("error encoding JSON: %s", err), http.StatusInternalServerError)
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Write(data)
+		if _, err := w.Write(data); err != nil {
+			http.Error(w, fmt.Sprintf("error writing data: %s", err), http.StatusInternalServerError)
```

Comment:
In fact, http.Error will call the function of `w.Write`. And that's meaningless. So no need to handle the error here.